build.sbt

```diff
@@ -58,7 +58,7 @@ val googleApiServicesPubsubVersion = s"v1-rev20220904-$googleClientsVersion"
 val googleApiServicesStorageVersion = s"v1-rev20240311-$googleClientsVersion"
 // beam tested versions
 val zetasketchVersion = "0.1.0" // sdks/java/extensions/zetasketch/build.gradle
-val avroVersion = "1.8.2" // sdks/java/extensions/avro/build.gradle
+val avroVersion = avroCompilerVersion // sdks/java/extensions/avro/build.gradle
```

Comment:
where does `avroCompilerVersion` get declared? 👀

Reply:
This is exposed by the [sbt-avro](https://github.com/sbt/sbt-avro) plugin. It makes sure we have the compiler and library version in sync